src/test/java/org/jenkinsci/plugins/testresultsanalyzer/ResultStatusTest.java

```diff
@@ -0,0 +1,15 @@
+package org.jenkinsci.plugins.testresultsanalyzer;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+public class ResultStatusTest {
+    @Test
+    void testGetValue() {
+        assertEquals("PASSED", ResultStatus.PASSED.getValue());
+        assertEquals("FAILED", ResultStatus.FAILED.getValue());
+        assertEquals("SKIPPED", ResultStatus.SKIPPED.getValue());
+        assertEquals("N/A", ResultStatus.NA.getValue());
+    }
```

Comment:
While technically correct, this test is superfluous since it does not test anything interesting. It would make much more sense to test a class that uses the `ResultStatus` and verify the `getValue` there as well.

Reply:
**Should I make a different PR for the implementation?**

Reply:
The original code `getCustomStatus` looks a little bit strange (as the whole code in this plugin;-). 

I would recommend that you set the method  `getCustomStatus` package private and test this method instead. There you will automatically test the enum as well.

Reply:
Hey @uhafner 
I didn't clearly understand what you meant about setting the method

Reply:
The test is asserting that the Java compiler is performing the operations that it is expected to perform.   Additional tests are welcomed as well, but I plan to merge this test because it is a step in the right direction.

This type of test executes very rapidly.  Thanks!